reverb/cc/platform/default/repo.bzl

```diff
@@ -219,17 +223,20 @@ filegroup(
 def _tensorflow_solib_repo_impl(repo_ctx):
     tf_lib_path = _find_tf_lib_path(repo_ctx)
     repo_ctx.symlink(tf_lib_path, "tensorflow_solib")
+    suffix = "so.2"
```

Comment:
can you set a PLATFORM variable at the top of the bzl and use that here?

Reply:
I don't know how to init PLATFORM in bzl.

Reply:
I added a function
```
def is_darwin(ctx):
    if ctx.os.name.lower().find("mac") != -1:
        return True
    return False
```